### PR TITLE
Added AudioMixing option

### PR DESCRIPTION
**Pull request by alexgritton**
Added audioMixing option to allow this audio to be mixed with other audio sources without pausing or changing their volume.

### CODE REVIEW

**Comment by Znegl** [comment]
Any news on this pull request?

**Comment by bradmartin** [comment]
I'll try to take a review of it tomorrow, sorry for the lack of movement, been swamped with getting NS7 support out for large number of users updating.

**Review by bradmartin — COMMENTED** [review]
Looks like if you can revert back the package rename and url change in the package.json this might be able to be merged. @alexgritton - if you are still interested in having it merged let me know, otherwise I'll close this PR and add the feature. Thanks for the work here.

File: src/android/player.ts [hunk 1]
```diff
@@ -83,8 +83,11 @@ export class TNSPlayer implements TNSPlayerI {
         }
 
         // request audio focus, this will setup the onAudioFocusChangeListener
-        this._mAudioFocusGranted = this._requestAudioFocus();
-        TNS_Player_Log('_mAudioFocusGranted', this._mAudioFocusGranted);
+        if (!options.audioMixing) {
+          this._mAudioFocusGranted = this._requestAudioFocus();
+          TNS_Player_Log('_mAudioFocusGranted', this._mAudioFocusGranted);
```
Comment (bradmartin) [inline src/android/player.ts:88]: Remove the player_log line and it'll resolve the conflict.

File: src/package.json [hunk 2]
```diff
@@ -38,9 +38,12 @@
       "git add"
     ]
   },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com/"
+  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/bradmartin/nativescript-audio.git"
+    "url": "https://github.com/alexgritton/nativescript-audio.git"
```
Comment (bradmartin) [inline src/package.json:46]: revert these and the PR should be able to be merged.

**Comment by Znegl** [comment]
> I'll try to take a review of it tomorrow, sorry for the lack of movement, been swamped with getting NS7 support out for large number of users updating.

Great! Thanks!

**Comment by alexgritton** [comment]
I'll clean this up today and get it ready to merge. Thanks for the response.

**Review by bradmartin — APPROVED** [review]

**Comment by bradmartin** [comment]
`6.1.0` published with the feature 💯 do let us know if any issues come up.

**Pull request merged**